### PR TITLE
Bump kinto-http

### PR DESCRIPTION
**Pull request by glasserc**
Pull in the fix to Kinto/kinto-http.js#180 so that we can address #421.

### CODE REVIEW

**Comment by glasserc** [comment]
@n1k0: Do you know if this is the correct way to use a library that is async/await?

**Comment by glasserc** [comment]
And also, can you tell me why the build is failing, even though `babel-polyfill` is part of the `devDependencies` 😆

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import "babel-polyfill";
```
Comment (Natim) [inline src/index.js]: I wish we could handle this without having to add babel-polyfill back in the mix.

Comment (n1k0) [inline src/index.js]: I don't clearly get why we suddenly need this, please refresh my memory

Comment (Natim) [inline src/index.js]: I guess it is an attempt to fix the tests, also it doesn't so maybe you know what is missing.

Comment (n1k0) [inline src/index.js]: fwiw `async/await` is supported through some Babel directibe added to `.babelrc` directly, no need for dedicated imports. At least that's what worked in kinto-http.js.

**Comment by n1k0** [comment]
Superseded by #423.

File: src/index.js (hunk 1 above)
Comment (glasserc) [inline src/index.js]: I'm not an expert, but it seems like the kinto-http.js artifacts did not include async/await code, but were already transformed into something that relied on a `regeneratorRuntime`. This `regeneratorRuntime` was provided by `babel-polyfill` or the "regenerator runtime", whatever that was.

**Pull request closed without merging**